base/src/org/compiere/process/EMailTest.java

```diff
@@ -52,9 +51,19 @@ protected String doIt () throws Exception
 	{
 		MClient client = MClient.get (getCtx(), p_AD_Client_ID);
 		log.info(client.toString());
-		
+
 		//	 Test Client Mail
-		String clientTest = client.testEMail();
+		int tableId = getTable_ID();
+		String clientTest = "";
+		if (tableId == I_AD_Client.Table_ID) {
+			clientTest = client.testEMail();
+		} else if (tableId == I_AD_OrgInfo.Table_ID) {
+			System.out.println("getRecord_ID() = " + getRecord_ID());
+
+//			MOrgInfo = new Query(getCtx(), I_AD_OrgInfo.Table_Name, I_AD_OrgInfo.COLUMNNAME_AD_Org_ID)
+//			MOrgInfo mOrgInfo = new MOrgInfo(getCtx(), getRecord_ID(), get_TrxName());
+//			String ret = EMail.validateMailDelivery(getCtx(), getRequestEMail(), getAD_EMailConfig_ID(), getName());

```

Comment:
@rcapecce If a code is not used, it is better not to add code even if they are comments, on the other hand, using System.out.println to report any message is not recommended, for this, it is better to use a log INFO type.

If your changing to this class doesn't add any real functionality, does it make sense to modify it?

Reply:
@e-Evolution I delete this section of commented code